File: android/app/src/main/java/dev/deliteai/assistant/domain/features/tts/TTSService.kt
/*
 * SPDX-FileCopyrightText: (C) 2025 DeliteAI Authors
 *
 * SPDX-License-Identifier: Apache-2.0
 */

package ai.nimbleedge.nimbleedge_chatbot.domain.features.tts

object TTSService {
    suspend fun getPCM(input: String): ShortArray = TTSManager.getPCM(input)
}

File: ios/NimbleEdgeAI/feature/tts/TTSService.swift
/*
 * SPDX-FileCopyrightText: (C) 2025 DeliteAI Authors
 *
 * SPDX-License-Identifier: Apache-2.0
 */

import Foundation

class TTSService {
    static func passLexiconToTheWorkflowScript(lexiconJson: [String:Any]) throws {
        try TTSManager.passLexiconToTheWorkflowScript(lexiconJson: lexiconJson)
    }

    static func getPCM(input: String) throws ->[Float] {
        try TTSManager.getPCM(input:input)
    }
}
